index.html

```diff
@@ -30,7 +30,7 @@
   </section>
 
   <video autoplay muted loop id="backgroundVideo">
-    <source src="img/Blurry-party-action.mp4" type="video/mp4">
```

Comment:
I commented this out because my cpu wanted to crash with the background video. Forgot to un-comment it.